pkcs11/_pkcs11.pyx

```diff
@@ -1550,6 +1550,18 @@ cdef class lib:
         except StopIteration:
             return token
 
+    def wait_for_slot_event(self, blocking=False):
+        cdef CK_SLOT_ID slot_id
+        cdef CK_FLAGS flag = CKF_DONT_BLOCK
+
+        if blocking:
+            flag ^= CKF_DONT_BLOCK
```

Comment:
```
if not blocking:
   flags |= CKF_DONT_BLOCK
```

This makes the whole thing more lucid to read. Keeps the naming consistent with PKCS#11 etc.